### DISCUSSION TITLE
Loaders 'ammo_out' pin

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#9314] [Ideas and suggestions]
Saw this suggestion pop up a couple times, figure it's worth adding to the github.
`ammo_out` wiring pin for gun loaders, functions similarly to the fuel output on reactor.

Output based on item condition & number of items. 
IE, a full coilgun ammo box outputs 1, half full 0.5, empty 0. 
A railgun loader full of 5 shells outputs 5.

**Comment by MaaiG** [comment]
Yep! This will allow the transfer of information to the ammunition depot.

**Comment by UserC2** [comment]
Shouldn't this be closed because there is an AMMUNITION_OUT connection?
